build.gradle.kts

```diff
@@ -19,9 +21,9 @@ plugins {
     alias(libs.plugins.kotlinx.binaryCompatibilityValidator)
 }
 
-apply plugin: 'kotlinx.team.infra'
+apply(plugin = "kotlinx.team.infra")
```

Comment:
can it be moved to `plugins` block?

Reply:
No :( There's no [plugin marker artifact](https://docs.gradle.org/8.8/userguide/plugins.html#sec:plugin_markers).